src/components/search-section/search-section.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Container from '../container/container';
+import styles from './styles.module.scss';
+
+import SearchIcon from '../../assets/search-icon.png';
+import PrimaryButton from '../primary-button/primary-button';
+
+
+
+const SearchSection = () => {
+  const [inputText, setInputText] = useState('');
+
+  const handleClick = (e) => {
+    setInputText(e.target.value);
+    // console.log(inputText);
+  }
+
+  return (
+    <div className={styles.sectionWrapper}>
+      <Container>
+        <section className={styles.section}>
+          <div className={styles.searchBox}>
+            <input
```

Comment:
input;'ą kartu su label sukurkite kaip atskirą komponentą, dažniausiai šis komponentas vadinamas Field, TextField, SelectField ir t.t.